### ISSUE TITLE
Scala support

### ISSUE THREAD

**Issue by viktorklang** [cloudstateio/cloudstate#16]
Add support for creating Scala-based stateful serverless functions

**Comment by janory** [comment]
Hi!
I would be interested about to make this!

**Comment by viktorklang** [comment]
@janory Hi! Have a go at that then :)

**Comment by viktorklang** [comment]
@janory It should be possible to base a Scala API on the work done in the `java-support`, let me know if you want to tackle this.

**Comment by janory** [comment]
@viktorklang Yes, happily. But I think I will need some help to start this. Can we have a chat sometime?

**Comment by viktorklang** [comment]
@janory Sure, what's your availability like?

**Comment by viktorklang** [comment]
@janory Now there's a Gitter community: https://gitter.im/CloudState-IO/community

**Comment by janory** [comment]
> @janory Now there's a Gitter community: https://gitter.im/CloudState-IO/community

Cool! I've joined! :)

**Comment by andyczerwonka** [comment]
@janory has there been any progress on this? Shall we collaborate?

**Comment by janory** [comment]
> @janory has there been any progress on this? Shall we collaborate?

@andyczerwonka No, not much. I wanted to have a kick-off call on this, but it haven't happened yet.
I would be happy to collaborate, let's have a call with the two of us and define some tasks and goals.
We could have a call on today, tomorrow or Saturday from 19 o'clock. (I am in GMT+2).

Let me know what works the best for you! :)

**Comment by viktorklang** [comment]
First call is next week :)
Feel free to start whenever, we'll be happy to provide any guidance/assistance needed ^^
